sample/HttpTrigger-CustomRoute/function.json

```diff
@@ -0,0 +1,16 @@
+﻿{
+    "bindings": [
+        {
+            "type": "httpTrigger",
+            "name": "req",
+            "direction": "in",
+            "methods": [ "get" ],
+            "route": "node/products/{category:alpha}/{id:int?}"
```

Comment:
Demonstrates an optional binding - the function returns a singleton or array based on this